src/renderers/opengl/app.rs

```diff
@@ -129,6 +137,8 @@ where
             })
         };
 
+
+        #[cfg(not(target_os = "macos"))]
```

Comment:
So there's literally no way that MacOS supports debug callbacks ? D:

Reply:
Nope

Reply:
Such a shame...